components/transfer/Transfer.vue

```diff
@@ -1,208 +1,237 @@
 <template>
-  <section>
-    <Loader v-model="isLoading" :status="status" />
-    <nuxt-link
-      v-if="$route.query.target"
-      :to="`/${urlPrefix}/u/${correctAddress}`"
-      class="pl-4 is-flex is-align-items-center">
-      <NeoIcon icon="chevron-left" class="mr-2" />
-      {{ $t('teleport.artistProfile') }}
-    </nuxt-link>
-    <p class="title is-size-3">
-      {{ $t('transfer') }} {{ unit }}
-      <span class="has-text-primary">${{ getCurrentTokenValue(unit) }}</span>
-    </p>
-
-    <NeoField>
-      <Auth />
-    </NeoField>
-
-    <div v-if="targets && hasBlockExplorer" class="mb-3">
-      {{ $t('teleport.donationSentTo') }}
-      <a
-        v-for="target in targets"
-        :key="target"
-        :href="addressExplorerUrl(target)"
-        target="_blank"
-        rel="nofollow noopener noreferrer"
-        class="has-text-weight-bold">
-        <Identity ref="identity" :address="target" show-onchain-identity />
-      </a>
+  <div class="transfer-card py-8 px-6">
+    <div
+      class="is-flex is-justify-content-space-between is-align-items-center mb-2">
+      <p class="has-text-weight-bold is-size-3">
+        {{ $t('transfer') }} {{ unit }}
+      </p>
+      <NeoDropdown position="bottom-left" append-to-body :mobile-modal="false">
+        <template #trigger="{ active }">
+          <NeoButton
+            icon="ellipsis-vertical"
+            class="square-32 mr-3"
+            :active="active" />
+        </template>
+
+        <NeoDropdownItem
+          v-clipboard:copy="generatePaymentLink(accountId)"
+          @click="toast(`${$i18n.t('toast.urlCopy')}`)">
+          {{ $t('transfers.payMeLink') }}
+        </NeoDropdownItem>
+      </NeoDropdown>
     </div>
 
-    <div class="is-flex is-align-items-center">
-      <NeoField>
-        {{ $t('general.balance') }}
+    <div class="is-flex mb-2">
+      <div class="token-price py-2 px-4">
+        {{ unit }} ${{ currentTokenValue }}
+      </div>
+    </div>
+
+    <div class="is-flex is-justify-content-space-between">
+      <div class="is-flex is-flex-direction-column">
+        <span class="has-text-weight-bold is-size-6 mb-1">{{
+          $t('transfers.sender')
+        }}</span>
+        <Auth />
+      </div>
+      <div class="is-flex is-flex-direction-column is-align-items-end">
+        <span class="has-text-weight-bold is-size-6 mb-1">{{
+          $t('general.balance')
+        }}</span>
         <Money :value="balance" inline />
-      </NeoField>
+        <span class="has-text-grey">≈ ${{ balanceUsdValue }}</span>
+      </div>
+    </div>
+
+    <hr class="my-5" />
```

Comment:
hr already has my-5 "built-in"
```suggestion
<hr/>
```